<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Treap Visual Fusion</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="ZoomGraphicsView" name="graphicsView">
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::ScrollHandDrag</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="controlPanel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="title">
       <string>Panel de Control</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelKey">
           <property name="text">
            <string>Clave (Key):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="keyLineEdit">
           <property name="placeholderText">
            <string>Ej: 10</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutBtns">
         <item>
          <widget class="QPushButton" name="insertButton">
           <property name="text">
            <string>Insertar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchButton">
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="text">
            <string>Eliminar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutAdv">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutSplit">
           <item>
            <widget class="QLineEdit" name="splitKeyLineEdit">
             <property name="placeholderText">
              <string>Split Key (Opcional)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="splitButton">
             <property name="text">
              <string>SPLIT (Seleccionado)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="joinButton">
           <property name="text">
            <string>JOIN (Unir Seleccionados)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutMeta">
         <item>
          <widget class="QPushButton" name="createTreapButton">
           <property name="text">
            <string>+ Nuevo Treap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteTreapButton">
           <property name="text">
            <string>Borrar Treap Vac√≠o</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Listo.</string>
      </property>
      <property name="styleSheet">
       <string>font-weight: bold; color: #333;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>ZoomGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
